models/vrp.rb

```diff
@@ -174,6 +174,11 @@ def self.check_consistency(hash)
         raise OptimizerWrapper::DiscordantProblemError, 'All matrices should have at least maximum(point[:matrix_index]) number of rows and columns' if matrix_not_big_enough
       end
 
+      # services consistency
+      if [hash[:services], hash[:shipments]].compact.flatten.any?{ |s| s[:minimum_lapse] && s[:maximum_lapse] && s[:minimum_lapse] > s[:maximum_lapse] }
```

Comment:
Can't we allow minimum and maximum lapse to be equal ?
```suggestion
      if [hash[:services], hash[:shipments]].compact.flatten.any?{ |s| s[:minimum_lapse] && s[:maximum_lapse] && s[:minimum_lapse] >= s[:maximum_lapse] }
```

Reply:
We can not : most of our customers using scheduling use _minimum_lapse_ = _maximum_lapse_ . This means that lapse should be exactly equal to this value. Otherwise lapse can be infinite.